Dockerfile

```diff
@@ -11,6 +11,9 @@ COPY remote-site-setup.sh /etc/osg/remote-site-setup.sh
 # can be dropped when provided by upstream htcondor-ce packaging
 COPY 51-gratia.conf /usr/share/condor-ce/config.d/51-gratia.conf
 
+RUN mkdir -p /etc/condor-ce/bosco_override
```

Comment:
I imagine that we'll create and package this dir in the upstream package so this can be dropped then, too!